tests/extendrtests/src/rust/Cargo.toml

```diff
@@ -16,6 +16,10 @@ crate-type = ["staticlib"]
 [dependencies]
 extendr-api = { version = "*", features = ["graphics", "ndarray", "either"] }
```

Comment:
I am not sure, but can we just enable `use_r_altlist` here? It is a test project, I doubt we need to ensure that we have R 4.3. Then you could simplify a lot of code that enables this feature.

Reply:
These things don't mix. It is not possible to enable compile-time cfg within cargo features.
Cargo features are cfg, with `cfg(feature = some_cargo_feature)`.

Reply:
Thanks for explaining, did not know that. Then LGTM

Reply:
I too was wondering if it's really not possible to do this in some simple way, but I couldn't come up with any nice solution. Even if extendr expose the feature gate, AltList-related APIs doesn't exist in the bindings of libR-sys, so extendr would need to provide dummy implementation for R < 4.3, which doesn't sound very useful either.

I think this problem (switching implementation based on the R version) is a general one and I need to find some solution at some point. But, I gave up in this pull request.